<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BBMonitorWidget</class>
 <widget class="QWidget" name="BBMonitorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BBMonitorWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCustomPlot" name="PlotWidgetY" native="true"/>
       </item>
       <item>
        <widget class="QCustomPlot" name="PlotWidgetW" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCustomPlot" name="PlotWidgetZ" native="true"/>
       </item>
       <item>
        <widget class="QCustomPlot" name="PlotWidgetX" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCustomPlot" name="PlotWidgetSum" native="true"/>
       </item>
       <item>
        <widget class="QCustomPlot" name="PlotWidgetMassCenter" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
